packages/editor/src/components/properties/AmbientLightNodeEditor.tsx

```diff
@@ -28,7 +28,7 @@ import React from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
-import { AmbientLightComponent } from '@etherealengine/spatial/src/renderer/components/AmbientLightComponent'
+import { AmbientLightComponent } from '@etherealengine/spatial/src/renderer/components/lights/AmbientLightComponent'
```

Comment:
Let's keep the components where they were

Reply:
They are in the same place but it's easier to find the lights now, is there a reason why we don't separate out the lights from the rest of the renderer?

Reply:
I suppose it's fine. Just wanted to keep the folder structure simple.